UnityProject/Assets/Scripts/ConfigurationSaves/ConfigurationSaves.cs

```diff
@@ -0,0 +1,294 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace ConfigurationSaves
+{
+	//TODO Directory.CreateDirectory(Application.persistentDataPath + Path);
+	public enum AccessCategory
+	{
+		Config,
+		Data,
+		Logs
+	}
+
+
+	public static class AccessFile
```

Comment:
class should probably just be ``File`` and there should be another class for ``Dir``

Reply:
well, you shouldn't have to have to mess around with directories , with this
it just makes them automatically that's the idea

Reply:
okay, fair.